src/explore-education-statistics-admin/src/pages/admin-dashboard/components/ManagePublicationsAndReleasesTab.tsx

```diff
@@ -138,6 +140,7 @@ const ManagePublicationsAndReleasesTab = ({
 
   useEffect(() => {
     if (selectedThemeAndTopic.topic.id) {
+      setLoading(true);
```

Comment:
Shouldn't we be changing the `loading` state back to false?

Reply:
It's not because the LoadingSpinner only shows if `themes && selectedThemeAndTopic && myPublications` evaluates to false.

I think Lewis has done it like this, because if you wrap everything in the loading spinner, you get the "You have no permissions text" flash up before the theme and topics have loaded in.

I think the solution is to not have as many useEffect calls, but will probably take some time to refactor.

Reply:
I think for now I'll just remove the loading state altogether